apps/livreGUI/animationController/AnimationController.cpp

```diff
@@ -38,6 +38,8 @@ struct AnimationController::Impl
         , _onFirstFrame( true )
     {
         _ui.setupUi( _animationController );
+        _frame.setUpdatedFunction( [&]
+            { emit _animationController->newFrameReceived(); });
```

Comment:
well, in theory the Qt signals can help, but:
- The signals are not (yet) emitted with binary serialization (used here), so we have to wait on Jafet's work
- The received values cannot be mapped directly to the Qt UI elements because the types differ (int32_t vs int or bool), there are boundar checks, etc. So, each valueChanged signal has to be connected to a lambda which does the checks/conversion (basically splitting the code in onNewFrameReceived() into separate lambdas, which is a bit cleaner but doesn't save much code).